testing/tests/DevExpress.ui.widgets.editors/validatorIntegration.tests.js

```diff
@@ -93,6 +109,55 @@ var Fixture = Class.inherit({
         assert.strictEqual(editorValidationError.message, "Required", "Message should came from dxValidator");
     });
 
+    QUnit.test("Editor/Validator.reset should not validate null value", function(assert) {
+        const editors = [
+            "dxAutocomplete", "dxCalendar", "dxCheckBox", "dxDateBox",
+            "dxDropDownBox", "dxHtmlEditor", "dxLookup", "dxRadioGroup",
+            "dxRangeSlider", "dxSelectBox", "dxSlider", "dxSwitch",
+            "dxTagBox", "dxTextArea", "dxTextBox"];
+
+
+        editors.forEach(editor => {
+            const validationCallback = sinon.spy();
+            this.fixture.createInstance(editor, { }, {
+                validationRules: [{
+                    type: "custom",
+                    validationCallback
+                }]
+            }, false);
+
+            this.fixture.editor.reset();
+
+            assert.notOk(validationCallback.called, `validationCallback should not be called for ${editor} after ${editor}.reset`);
+
+            this.fixture.validator.reset();
```

Comment:
I suggest extracting to separate test cases.

Reply:
OK